### PR TITLE
Readme changes

### PR DESCRIPTION
**Pull request by ashleyharvey**

### CODE REVIEW

**Comment by ashleyharvey** [comment]
Proposed changes. Some of it may be too advanced, I'm not sure.
I'm a little nervous about removing the `-r` from `grep`; maybe I missed something but per my testing, `-r` represents recursion which is not applicable when dealing with a file directly as opposed to a directory. Also this PR addresses the `stat` issue linked in Issue-15.

**Comment by stevemao** [comment]
@ashleyharvey Thanks so much! The changes look good to me in general.

File: readme.md [hunk 1]
```diff
@@ -155,7 +161,7 @@ touch 'new file' # updates the file's access and modification timestamp if it al
 ```shell
 mkdir 'untitled folder'
 # or
-mkdir -p 'path/may/not/exist/untitled folder'
+mkdir -p 'path/may/not/exist/untitled\ folder'
```
Comment (stevemao) [inline readme.md]: I'm not sure why this change is needed?

File: readme.md [hunk 2]
```diff
@@ -232,16 +239,18 @@ rm -r my_useless_folder
 **STOP OPENING YOUR FINDER OR FILE EXPLORER** :-1:
 
 ```shell
-ls -la my_folder
+ls my_folder        # Simple
```
Comment (stevemao) [inline readme.md]: I don't mind adding this. But I actually rarely use it. I'm wondering if we can add some other flags that might be useful?

File: readme.md (hunk 1 above)
Comment (ashleyharvey) [reply readme.md]: Good to get in the habit of escaping spaces.

File: readme.md (hunk 2 above)
Comment (ashleyharvey) [reply readme.md]: Sure! I most often use `ls -alrth` to show a list of files, sorted by time (reversed - most recently edited at the bottom), with human-readable sizing.

File: readme.md [hunk 3]
```diff
@@ -332,7 +332,7 @@ cat apps/settings.py | less
 **STOP CMD/CTRL + F IN A FOLDER** :-1:
 
 ```shell
-grep -ir "Query" file.txt
+grep -i "Query" file.txt
```
[the remark this answers is not in the document]
Comment (stevemao) [reply readme.md]: CC @umr55766

[the remark this answers is not in the document]
Comment (umr55766) [reply readme.md]: Thanks for reaching out @ashleyharvey and @stevemao for tagging me.

TBH, I had added `flag i` and `flag r` because I use that almost always with grep command.
And I just checked my original PR and found that it was only an example (& command syntax was separate), you can find those here https://github.com/you-dont-need/You-Dont-Need-GUI/pull/33/files.

**Comment by cht8687** [comment]
👍  let's merge this improvement.
